.env.example

```
@@ -4,3 +4,6 @@ VITE_PUBLIC_POSTHOG_HOST=https://eu.i.posthog.com
 # `bun run build` or `bun run deploy`. Set it in the build environment for CI.
 # It identifies the project but cannot read Sentry data. Leave blank to disable it.
 VITE_PUBLIC_SENTRY_DSN=
+
+# evlog — wide event delivery
+SENTRY_DSN= # project DSN
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)